### PR TITLE
Makefile addition

### PR DESCRIPTION
**Pull request by NaveenGokavarapu19**
fix for issue  #106

### CODE REVIEW

File: development.md
Comment (rakshitgondwal) [inline development.md:1]: This file is not required in this PR. I guess you mixed up #109 with this.

File: Makefile [hunk 1]
```diff
@@ -25,4 +25,8 @@ test:
 	go test ./...
 
 verify:
-	@hack/verify.sh
\ No newline at end of file
+	@hack/verify.sh
+
+
+test-e2e: build
+	@hack/run-e2e.sh
```
Comment (rakshitgondwal) [inline Makefile:32]:
```suggestion

test-e2e: build
	@hack/run-e2e.sh
```

You can remove this extra line and add a new line at the bottom to remove the `No newline at the end of the file` error.

**Review by rakshitgondwal — COMMENTED** [review]
Left some comments :)

**Comment by NaveenGokavarapu19** [comment]
done the suggested change

**Comment by NaveenGokavarapu19** [comment]
closing the pull request according to suggestion of @rjsadow

**Pull request closed without merging**